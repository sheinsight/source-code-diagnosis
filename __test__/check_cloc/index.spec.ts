import path from "node:path";
import { checkCloc } from "../../index";
import { expect, test } from "vitest";

test("check_cloc", async () => {
	const res = await checkCloc({
		cwd: path.resolve(__dirname, "fixtures"),
		ignore: [],
	});

	expect(res.length).toBe(2);

	expect(res).toMatchSnapshot();
});
