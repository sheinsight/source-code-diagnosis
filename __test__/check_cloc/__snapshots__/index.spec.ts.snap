// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`check_cloc 1`] = `
[
  {
    "language": "CSS",
    "linesOfCode": "3",
  },
  {
    "language": "JavaScript",
    "linesOfCode": "7",
  },
]
`;
